### PR TITLE
Ship and creature pack I

### PR DESCRIPTION
**Pull request by Lamarr13**
Remplacement and add for creatures and ships
- Kotor 1 and 2 droids and creature models
- Onderonian ships + others
-Remplacement of leviathan/centurion models with better optimized one and looking ones
By Lamarr

### CODE REVIEW

File: swlor2_mdl/C_MK1DRD.mdl [hunk 1]
```diff
@@ -0,0 +1,16577 @@
+# NeverBlender 4.0.0|Blender 4.0.2|2024-01-04
```
Comment (zunath) [inline swlor2_mdl/C_MK1DRD.mdl:1]: Bunch of models in here need to be compiled

**Review by zunath — CHANGES_REQUESTED** [review]
Requesting changes above

File: swlor2_mdl/C_MK1DRD.mdl (hunk 1 above)
Comment (Lamarr13) [reply swlor2_mdl/C_MK1DRD.mdl]: Mdls compiled (dunno how I created a third empty commit)

Comment (zunath) [reply swlor2_mdl/C_MK1DRD.mdl]: Still not compiled. See this one as an example: https://github.com/zunath/SWLOR_Haks/blob/4f1ae99195d991db1fbd82f46ef56be89effec06/swlor2_mdl/C_Drexl.mdl

File: swlor2_2da/appearance.2da [hunk 2]
```diff
@@ -10220,4 +10220,42 @@
...
 10218   "[SWLOR] Ship - Armored Transport "                              ****         c_fl_artr                    c_fl_artr           ****             N           F           1             1                 1                1                NORM       1.0        2.0       0.3        0.5           1        0.3       1.8            H              1              23           1         1         po_zo_war01       3              9                33             55             1           60           30           head_g            5          1             
-10219   "(Dynamic) Ewok"                                                 ****         Character_model              D                   default          R           P           1             1                 1.5             1.5               NORM       1.06       2.12      0.3        0.5           1.5      0.3       1.4            H              1              169          1         1         ****              3              9                0              0              1           60           30           head_g            6          1
```
Comment (zunath) [inline swlor2_2da/appearance.2da:10223]: The new entry for Ewoks is getting removed. Please copy/paste it over to your file to make sure we don't lose this change.

**Review by zunath — APPROVED** [review]

**Pull request merged**